Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:14.04
+MAINTAINER Vaibhav Bhembre <vaibhav@digitalocean.com>
+
+ENV GOROOT /goroot
+ENV GOPATH /go
+ENV PATH $GOROOT/bin:$PATH
+ENV APPLOC $GOPATH/src/github.com/digitalocean/ceph_exporter
+
+ADD . $APPLOC
+
+RUN apt-get update && \
+    apt-get install -y build-essential git curl && \
+    apt-get install -y librados-dev librbd-dev
```

Comment:
No other reason than helping separate concerns/readability especially if we only need to modify or move the ceph/rados/rbd installation part without a performance penalty.